src/core/plugin/luapi_application.h

```diff
@@ -1960,9 +1962,11 @@ static int applib_changeToolColor(lua_State* L) {
         return luaL_error(L, "tool \"%s\" has no color capability", toolTypeToString(toolType).data());
     }
 
+    lua_pop(L, 3);
```

Comment:
We agreed not to pop directly before return statement, since it is unnecessary. See https://github.com/xournalpp/xournalpp/pull/4837#issuecomment-1657824620